server/lib/logger.js

```diff
@@ -41,8 +35,8 @@ Logger.sendMail = (errorCode, errorMessage, data) => {
   if (data) {
     mailOptions.attachments = [
       {
-        filename: 'data.json',
-        content: JSON.stringify(data),
```

Comment:
Has this been tested? Just want to make sure it works as expected and we get the actual correct object through?